[:]/trace.txt

```diff
@@ -0,0 +1,70 @@
+task_id	hash	native_id	name	status	exit	submit	duration	realtime	%cpu	peak_rss	peak_vmem	rchar	wchar
```

Comment:
Did you want to include this file or is it a leak?

Reply:
I don't know why it keeps being pushed its in the .gitignore:
https://github.com/MazdaX/nf-cage/blob/4ebad885f695396b4c0e81cf836a90f9645bf3a7/.gitignore#L31
I must have missed it during the conflict resolution step. It happens I guess ;-)